rust-toolchain.toml

```
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.81"
-components = ["rustfmt", "clippy"]
+channel = "stable-aarch64-apple-darwin"
```

```suggestion
channel = "1.86"
```

Specifying the current channel means Drink! will only compile on Apple Silicon.

I'm actually not sure why Drink! is requiring a specific toolchain like this, do you? If not we can look into that later. 

For now, ink! v6 requires `rustc >= 1.86`, so `1.86` seems like a reasonable choice before we make a decision on the rest (i.e. the relevance of specifying a channel this way).

---

This is a mistake when I tested it on local, thanks for pointing it out. Resolved in latest commit.